Add identify device feature to keithley 2410 and 2510

rowan04 opened the pull request:
- used through the api with a toggle
- shows an identify message on device screen
- add the device identifier code onto the device display when it's identified
- some code improvements (e.g. whitespace removal)
- add width tag in css for toggle-label which aligns all the toggles together

resolves #2

rowan04 committed:
Add identify device feature to keithley 2410 and 2510

- used through the api with a toggle
- shows an identify message on device screen

rowan04 committed:
Refactor identify feature code, and add ident string to device display when identified

- moved identify feature code into gpibdevice.py
- add the device identifier code onto the device display when it's identified
- some code improvements (e.g. whitespace removal)

rowan04 committed:
Align toggles with CSS change

- add width tag in css for toggle-label
- which aligns all the toggles together

timcnicholls commented on src/odin_gpib/keithley2410.py:
```diff
@@ -27,11 +27,13 @@ def __init__(self, device, ident, lock):
-        self.ramping_flag = False 
+        self.device_control_enable = True
+        self.ramping_flag = False
+        self.write(':DISP:WINDOW1:TEXT:STAT 0')
```
Can these calls to reset the display text on initialisation in both 2410 & 2510 classes be refactored into the base class, or call `self.set_identify(False)` ??

rowan04 replied:
addressed

timcnicholls reviewed (changes requested):
Looks good! Minor suggestion for a refactor to remove duplicate code

rowan04 committed:
Refactor resetting display text on initialisation

- code review suggestion
- makes use of self.set_identify(False)
- to reset the display text on initialisation
